src/config/schema.ts

```diff
@@ -95,7 +94,7 @@ export const chainsSchema = z
         ctx.addIssue({
           code: 'custom',
           message: 'Missing provider. At least one provider is required.',
-          path: ['chains', chainId, 'providers'],
+          path: [chainId, 'providers'],
```

Comment:
Before removing the redundant `'chains'` key, the configuration throws an error like the following when an error condition is met during the parsing of the full Airseeker config:
```ts
 {
      "code": "custom",
      "message": "Missing provider. At least one provider is required.",
      "path": [
        "chains",
        "chains",
        "31337",
        "providers"
      ]
    },
```

After its removal, the error path is corrected:
```ts
 {
      "code": "custom",
      "message": "Missing provider. At least one provider is required.",
      "path": [
        "chains",
        "31337",
        "providers"
      ]
    },
```